docs/source/tutorial_worlds.rst

```diff
@@ -249,3 +249,14 @@ for GPU-based models, which prefer to process more examples at a time.
 
 Tip: if you implement ``batch_act()``, your ``act()`` method can just call ``batchact()``
 and pass the observation it is supposed to process in a list of length 1.
+
+Multiprocessed Data Loading
+~~~~~~~~~~~~~~
```

Comment:
this should extend the width of the text above it

Reply:
i would put pytorch data loader in the title as well perhaps?